C-Plus-Plus/dp/numberOf_subsets_with_given_diff.cpp

```diff
@@ -0,0 +1,61 @@
+// Program to find number of subsets with given difference between their sums
+// Uses dynamic programming to solve the problem. We'll use the existing problem that counts subsets with a given sum.
```

Comment:
- Make your file name concise and readable `numberOf_subsets_with_given_diff.cpp`
- Add a beginner-friendly description at the top 
- Add a new line after description